tools/clang/test/HLSL/shader_cvc_mismatch.hlsl

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -fsyntax-only -ffreestanding -verify %s
+
+
+[shader("compute")]
+[shader("vertex")]
+[shader("compute")]
```

Comment:
Should we allow two `shader("compute")` attributes and nothing else? That seems weird, but valid.